pages/discover.js

```diff
@@ -93,10 +92,10 @@ export default function Discover() {
             Managing the data where it is created.
           </h3>
           <div className="pt-4">
-            CDS avoids moving data to centralized infrastructures by computing it directly where it
-            is created, reducing this way the data transmission efforts. CDS can select the piece of
-            data to share with indexing servers, limiting it, for example, to data intended to
-            produce analyses or commercial services.
+            CDS avoids the use of centralized infrastructures by computing data directly where it is
+            created, thus reducing risks and cost du to data transport. To do so, CDS is able to
+            select the specific piece of data to share with indexing servers, limiting it, for
```

Comment:
With indexing servers, or to synchronize with other actors - to be reworked.

For all the section we should focus on the advantages of doing things locally - the user has the control on his data, its available even offline, and the calculation power to execute it at a different place can be avoided as the device directly execute the computation of data.